src/main/java/org/sagebionetworks/bridge/models/permissions/PermissionDetail.java

```diff
@@ -0,0 +1,62 @@
+package org.sagebionetworks.bridge.models.permissions;
+
+import org.sagebionetworks.bridge.models.accounts.AccountRef;
+
+import java.util.Objects;
+
+public final class PermissionDetail {
+    
+    private final String guid;
+    private final String userId;
+    private final AccessLevel accessLevel;
+    private final EntityType entityType;
+    private final String entityId;
+    private final AccountRef userAccountRef;
+    
+    public PermissionDetail(Permission permission, AccountRef userAccountRef) {
```

Comment:
Consider adding entityName, so that users can figure out what the permissions are for without looking up a guid.

Reply:
Added an EntityRef to PermissionDetail that includes type, id, and name in case further entity details are needed later.